package.json

```diff
@@ -59,9 +62,11 @@
     "geostats": "1.7.0",
     "gm": "1.23.1",
     "imports-loader": "0.8.0",
+    "jest": "^24.8.0",
     "loader-utils": "1.2.3",
     "merge": "1.2.1",
     "mini-css-extract-plugin": "0.5.0",
+    "mobile-detect": "^1.4.3",
```

Comment:
Take out the ^ as it's not used in the others and change this https://github.com/oskariorg/oskari-frontend/blob/1.52.0/webpack/oskari-core.js#L7 to use the version from npm instead of the saved on under libraries. The one under libraries can be removed after this. Maybe even split this change to another pr?

Reply:
I will do this change in another pr.

Reply:
I decided to include these changes to this pr in spite of my previous comment.